angular.json

```diff
@@ -86,20 +86,20 @@
               ],
               "fileReplacements": [
                 {
-                  "replace": "web-frame/src/app-shared/config/defaults/available-feature-modules.ts",
-                  "with": "web-frame/src/app-shared/config/defaults/available-feature-modules.prod.ts"
+                  "replace": "web-frame/src/app-shared/config/replacements/available-feature-modules.ts",
+                  "with": "web-frame/src/app-shared/config/replacements/available-feature-modules.prod.ts"
                 },
                 {
-                  "replace": "web-frame/src/app-shared/config/defaults/environment.ts",
-                  "with": "web-frame/src/app-shared/config/defaults/environment.prod.ts"
+                  "replace": "web-frame/src/app-shared/config/replacements/environment.ts",
+                  "with": "web-frame/src/app-shared/config/replacements/environment.prod.ts"
                 }
               ]
             },
             "proxy": {
               "fileReplacements": [
                 {
-                  "replace": "web-frame/src/app-shared/config/defaults/environment.ts",
-                  "with": "web-frame/src/app-shared/config/defaults/environment.proxy.ts"
+                  "replace": "web-frame/src/app-shared/config/replacements/environment.ts",
+                  "with": "web-frame/src/app-shared/config/replacements/environment.proxy.ts"
```

Comment:
@mcristi, there were to many conflicts created by moving the configuration from `~app-shared/config` into `~app-shared/core`. And because that module wasn't removed, I renamed the `defaults` directory to `replacements`, not `configs`.